docs/modules/rust/quickstart.md

```diff
@@ -239,7 +243,7 @@ spacetime publish --project-path server quickstart-chat
 You can use the CLI (command line interface) to run reducers. The arguments to the reducer are passed in JSON format.
 
 ```bash
-spacetime call quickstart-chat send_message 'Hello, World!'
+spacetime call quickstart-chat send_message "Hello, World!"
```

Comment:
Why is this change necessary?

Reply:
I don't particularly love this, but since this still works in `bash`, and since this is a quickstart, I think it's reasonable to make this quickstart support other shells (even if we don't commit to supporting them everywhere).